packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/emailtemplate.md

```diff
@@ -0,0 +1,11 @@
+If you're using the Sign-In Widget or SDK sample app use `https://localhost:44314/magiclink/callback?otp=${oneTimePassword}&state=${request.relayState}`. The following snippet shows how the URL looks in the HTML message:
```

Comment:
```suggestion
If you're using the Sign-In Widget or an SDK sample app use `https://localhost:44314/magiclink/callback?otp=${oneTimePassword}&state=${request.relayState}`. The following snippet shows how the URL looks in the HTML message:
```

Reply:
Changed this to "If you're using the Sign-In Widget sample app or the SDK sample app ..." as there are only two sample apps altogether.